docs/byoh_agent.md

```diff
@@ -0,0 +1,116 @@
+# BYOH Agent
+
+BYOH agent is a binary that runs on the hosts and its responsibility include -  
+1. Registration - Register the host to the cluster capacity pool
+2. Setup - Install / Uninstall kubernetes components on the host
+3. Bootstrap - Convert the host to a kubernetes node using kubeadm
+
+## Usage of BYOH agent
+
+Below flags are supported by the agent:-  
+```
+--downloadpath string 
+```
+File System path to keep the downloads (default `/var/lib/byoh/bundles`)  
+<br />
```

Comment:
This is not required, there is an extra space added because of it. Applicable for all "<br\>" tags
```suggestion

```